doc/user-guide.md

```diff
@@ -249,6 +249,90 @@ return reconcile.Result{RequeueAfter: time.Second*5}, nil
 
 **Note:** Returning `Result` with `RequeueAfter` set is how you can periodically reconcile a CR.
 
+#### Reconcile Result Use Cases
+**The following are possible reconcile loop return options.**
+
+#### 1. With the error:
+
+If an error is encountered during processing the appropriate return option is to return an error.
+This results in the reconcile loop being re-triggered to run again.
+
+Usage
+```Go
+return reconcile.Result{}, err
+```
+
+Example:
+
+In the [`memcached_controller.go`][memcached_controller] example below a `reconcile.Result{}, err` is used when there is an error reading the object.
```

Comment:
```suggestion
In the example below a `reconcile.Result{}, err` is used when there is an error reading the object.
```
IHMO  remove the `[`memcached_controller.go`][memcached_controller] ` and would be better in order to keep it maintained and try to be less verbose.

Reply:
Sure. Fixed.